.travis.yml

```diff
@@ -1,15 +1,43 @@
-sudo: required
-dist: trusty
-language: python
-python:
-    - "2.6"
-    - "2.7"
-    - "3.3"
-    - "3.4"
-    - "3.5"
-    - "3.6"
+matrix:
+  include:
+    - os: linux
+      language: python
+      env: MINICONDA="https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh" PYTHONVER="2.7"
+    - os: linux
+      language: python
+      env: MINICONDA="https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh" PYTHONVER="3.5"
```

Comment:
does miniconda not support 3.3 and 3.4 or just choosing to not support older versions?